<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="convertData">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert Data</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textEntry">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="displayLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>File Path</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="textboxText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>361</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="locateFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Output Directory</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>530</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
